pylettize/__init__.py

```diff
@@ -1,5 +1,6 @@
 """Pylettize package."""
 
-from .pylettize import main
+from . import cli as cli
+from . import pylettize as core
```

Comment:
I've been considering renaming the main pylettize module to `core`, but I'm not sure if now's the time to do so.